content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst

```diff
@@ -1,418 +1,166 @@
+   * -
+     - FIFO
+     - LIFO
+     - FEFO
+     - Closest Location
+     - Least Packages
+   * - Required features
+     - Lots & Serial Numbers
+     - Lots & Serial Numbers
+     - Lots & Serial Numbers, Expiration Date
+     - Storage Locations, Multi-Step Routes
+     - Packages
 
-Lot `000001` contains five shirts, lot `000002` contains three shirts, and lot `000003` contains two
-shirts.
+.. _inventory/removal/lots-setup:
 
-To see the removal strategy in action, go to the :menuselection:`Sales app` and click
-:guilabel:`Create` to create a sales order. Next, select a :guilabel:`Customer` from the drop-down
-menu. Then click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab. Select a product (for
-this example, the :guilabel:`White Shirt`) from the drop-down menu, or type in the name of the
-product in the field. Enter a quantity (for this example, `6.00`) in the :guilabel:`Quantity` field,
-then click :guilabel:`Save`, then click :guilabel:`Confirm`.
+Lots and serial numbers
+~~~~~~~~~~~~~~~~~~~~~~~
 
-Once the sales order is confirmed, the delivery order will be created and linked to the picking, and
-the oldest lot numbers will be reserved thanks to the :abbr:`FIFO (First In, First Out)` strategy.
-All five shirts from lot `000001` and one shirt from lot `000002` will be selected to be sent to the
-customer.
+Lots and serial numbers differentiate identical products and track information like arrival or
+expiration dates. To enable this feature, navigate to :menuselection:`Inventory --> Configuration
+--> Settings`. Under the :guilabel:`Traceability` heading, check the box beside :guilabel:`Lots &
+Serial Numbers` to enable the feature.
 
-.. image:: removal/reserved-lots-fifo-strategy.png
+.. image:: removal/enable-lots.png
    :align: center
-   :alt: Two lots being reserved for a sales order with the FIFO strategy.
-
-LIFO (Last In, First Out)
--------------------------
-
-The :abbr:`LIFO (Last In, First Out)` removal strategy works in the **opposite** manner from the
-:abbr:`FIFO (First In, First Out)` strategy. With this method, the products that are received
-**last** are moved out first. This method is mostly used for products without a shelf life, and no
-time-sensitive factors, such as expiration dates.
+   :alt: Enable lots and serial numbers.
 
-In this example, there are three lots of cinder blocks. The blocks are from the *All/Building
-Materials* category, where *FIFO* is set as the removal strategy. In the :guilabel:`Inventory
-Valuation Report`, the three different receipts are listed with the amounts.
+Next, ensure the intended product is tracked by lots or serial numbers by navigating to the product
+form through :menuselection:`Inventory --> Products --> Products`, and selecting the desired
+product. On the product form, switch to the :guilabel:`Inventory` tab, and under the
+:guilabel:`Tracking` field, select either the :guilabel:`By Unique Serial Number` or :guilabel:`By
+Lots` options.
```

Comment:
After selecting this on the product form, the next step is to add the lot/serial # to the product so Odoo can actually track it properly. It's not enough to just enable the setting, so this last step is what completes the instruction. 

Therefore, I'd briefly list the step(s) on how to do this (e.g. **☰ Lots/Serial Numbers** smart button on product form) or point to other documentation that has that information.